arduino/examples/ESP32SerialBridge/ESP32SerialBridge.ino

```diff
@@ -161,7 +161,7 @@ void setup() {
 #endif // PRINT_PACKETS
 
     // Add Serial1 as a bridge
-    openmrn.add_gridconnect_port(new Esp32HardwareSerialAdapter(Serial1));
+    openmrn.add_gridconnect_port(&Serial1);
```

Comment:
since this example no longer has hard dependencies on ESP32 should it be renamed to simply ArduinoSerialBridge?

Reply:
why would this be not specific to the ESP32?
- it has the ESP32 CAN
- it is using the fd-based configuration system that ESP32 has
- it is using ESP32 GPIO numbers and assignments.

Reply:
Ahh, forgot that it had the can GPIO pins and the SPIFFS VFS paths. Scratch that suggestion on moving to generic. Is there a need for a similar example for STM32?

Reply:
yes i'm going to make some stm32 examples.